test/ci/buildkite/ngtf-ie-modeltest.yml

```diff
@@ -0,0 +1,42 @@
+  - command: |
+      source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
+      python3 build_ngtf.py ${BUILD_OPTIONS} \
+      --artifacts /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID \
+      --use_tensorflow_from_location /localdisk/buildkite-agent/prebuilt_tensorflow_2_2_0
+      pip install -U /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/tensorflow/tensorflow-2.2.0-cp36-cp36m-linux_x86_64.whl
+      pip install -U /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/ngraph_tensorflow_bridge-*.whl
+
+    label: ":hammer_and_wrench: Build/Install ${NGRAPH_TF_BACKEND}"
+    timeout_in_minutes: 30
+    agents:
+      queue: ${BUILDKITE_AGENT_META_DATA_QUEUE}
+      name: ${BUILDKITE_AGENT_META_DATA_NAME}
+
+  - wait
+  - command: |
+      source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate
+      export SRCDIR=`pwd`
+      pip install Pillow
+      cd /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID
+      bash \$SRCDIR/test/ci/buildkite/run_model_infer.sh resnet_50 bike 'mountain bike, all-terrain bike'
+      bash \$SRCDIR/test/ci/buildkite/run_model_infer.sh resnet_50_v1.5 bike 'mountain bike, all-terrain bike'
+      bash \$SRCDIR/test/ci/buildkite/run_model_infer.sh resnet_50_v2 bike 'mountain bike, all-terrain bike'
+      bash \$SRCDIR/test/ci/buildkite/run_model_infer.sh resnet_v1_50 bike 'mountain bike, all-terrain bike'
+      bash \$SRCDIR/test/ci/buildkite/run_model_infer.sh resnet_v2_101 bike 'mountain bike, all-terrain bike'
+      bash \$SRCDIR/test/ci/buildkite/run_model_infer.sh resnet_v2_152 bike 'mountain bike, all-terrain bike'
+      bash \$SRCDIR/test/ci/buildkite/run_model_infer.sh resnet_v2_50 bike 'mountain bike, all-terrain bike'
+      bash \$SRCDIR/test/ci/buildkite/run_model_infer.sh inception_v3 bike 'mountain bike, all-terrain bike'
+      bash \$SRCDIR/test/ci/buildkite/run_model_infer.sh inception_v4 bike 'mountain bike, all-terrain bike'
```

Comment:
Can you create a run manifest for the models? It would be easier to track, enable/disable individual models and/or leave comments if we find any issues.